src/Language/Printer.php

```diff
@@ -86,379 +92,356 @@ public static function doPrint($ast)
+                return $this->addDescription($node->description, $this->join(
+                    [
+                        'enum',
+                        $this->p($node->name),
+                        $this->printList($node->directives, ' '),
+                        $this->printListBlock($node->values),
+                    ],
+                    ' '
+                ));
+            case $node instanceof EnumTypeExtensionNode:
+                return $this->join(
+                    [
+                        'extend enum',
+                        $this->p($node->name),
+                        $this->printList($node->directives, ' '),
+                        $this->printListBlock($node->values),
+                    ],
+                    ' '
+                );
+            case $node instanceof EnumValueDefinitionNode:
+                return $this->addDescription(
+                    $node->description,
+                    $this->join([$this->p($node->name), $this->printList($node->directives, ' ')], ' ')
+                );
+            case $node instanceof EnumValueNode:
+                return $node->value;
+            case $node instanceof FieldDefinitionNode:
+                $argStrings = array_map(function ($item) : string {
+                    return $this->p($item);
+                }, $this->listToArray($node->arguments));
+                $noIndent   = Utils::every($argStrings, static function (string $arg) : bool {
+                    return strpos($arg, "\n") === false;
+                });
+
+                return $this->addDescription(
+                    $node->description,
+                    $this->p($node->name)
+                    . ($noIndent
+                        ? $this->wrap('(', $this->join($argStrings, ', '), ')')
+                        : $this->wrap("(\n", $this->indent($this->join($argStrings, "\n")), "\n)"))
+                    . ': ' . $this->p($node->type)
+                    . $this->wrap(' ', $this->printList($node->directives, ' '))
+                );
+            case $node instanceof FieldNode:
+                return $this->join(
+                    [
+                        $this->wrap('', $node->alias->value ?? null, ': ') . $this->p($node->name) . $this->wrap(
+                            '(',
+                            $this->printList($node->arguments, ', '),
+                            ')'
+                        ),
+                        $this->printList($node->directives, ' '),
+                        $this->p($node->selectionSet),
+                    ],
+                    ' '
+                );
+            case $node instanceof FloatValueNode:
+                return $node->value;
+            case $node instanceof FragmentDefinitionNode:
+                // Note: fragment variable definitions are experimental and may be changed or removed in the future.
+                return sprintf('fragment %s', $this->p($node->name))
```

Comment:
```suggestion
                return 'fragment ' . $this->p($node->name)
```

I think that would be much more readable. (Full disclosure: I despise `sprintf` :shrug:)

Reply:
Agreed, thank you. e8340ae